scripts/check_formatting

```diff
@@ -42,18 +40,14 @@ find scripts -type f -exec shellcheck {} +
 
 # Fortunately, rustfmt has the --check option that will make it exit with 1
 # if formatting has to be applied.
-cargo fmt --all --manifest-path ./examples/Cargo.toml -- --check
-cargo fmt --all --manifest-path ./sdk/rust/Cargo.toml -- --check
-cargo fmt --all --manifest-path ./oak/server/rust/Cargo.toml -- --check
+cargo fmt --all --manifest-path ./Cargo.toml -- --check
```

Comment:
Now we can remove all these `manifest-path` flags, since the manifest is in the canonical location.

Reply:
Done